lib/guardian_db.ex

```diff
@@ -154,5 +148,9 @@ defmodule GuardianDb do
     end
   end
 
-  def repo, do: @repo
+  def repo do
```

Comment:
No longer read from the module attribute, we do not need to recompile when it change